images/wkdev_sdk/Containerfile

```diff
@@ -127,5 +127,8 @@ RUN export QT_VERSION=$(qmake6 -query QT_VERSION) && \
       ln -s ${directory} ${directory}/${QT_VERSION}  >/dev/null 2>&1 || true; \
     done
 
+# Build custom sccache version (the Ubuntu provided 0.7.7 package is broken)
+RUN cargo install --root /usr --version 0.8.1 --locked sccache
```

Comment:
I might prefer `/usr/local` but pleasantly easy to build. :+1:

Reply:
Indeed, thinking about it /usr isist is for system packages nothing self-built. I will adapt the PR.